CHANGELOG.adoc

```diff
@@ -25,6 +25,7 @@ More existing APIs will be migrated to this new common pattern.
  * `public Optional<IRepresentationEventProcessor> acquireRepresentationEventProcessor(IRepresentationConfiguration configuration, IInput input)`;
  * `Flux<IPayload> getSubscription(String editingContextId, IRepresentationConfiguration representationConfiguration, IInput input);`
  - https://github.com/eclipse-sirius/sirius-web/issues/3623[#3623] [form] Remove `IWidgetSubscriptionManagerFactory`
+ - https://github.com/eclipse-sirius/sirius-web/issues/3634[#3634] [sirius-web] Remove `GraphQLNodeStyleFragment` from `NodeTypeRegistry`, you can specify additional fields to be retreived by the subscription using the `DocumentTransform` GraphQL API and the `apolloClientOptionsConfigurersExtensionPoint` extension point
```

Comment:
I'll just mention that the other parts of the API used to contribute custom nodes will also be removed soon such as:

- nodeLayoutHandlers
- nodeConverters
- nodeTypeContributions

Since they will be replaced with the extension registry too

Reply:
I'll also talk about the fact that all existing customization of the GraphQL queries, subscriptions or mutations and future ones will all have to rely on this API too.